settings.php

```diff
@@ -230,16 +230,24 @@
         saml2_settings::OPTION_DUAL_LOGIN_NO      => get_string('no'),
         saml2_settings::OPTION_DUAL_LOGIN_YES     => get_string('yes'),
         saml2_settings::OPTION_DUAL_LOGIN_PASSIVE => get_string('passivemode', 'auth_saml2'),
+        saml2_settings::OPTION_DUAL_LOGIN_TEST    => get_string('test_idp_conn', 'auth_saml2')
```

Comment:
Could we add a trailing comma to the end of this multiline array please.

> Note the last item has a trailing comma left there which allows to extend the list of items later with a cleaner diff. For the same reason, it is better not to align the assignment operators. 
https://docs.moodle.org/dev/Coding_style#Wrapping_arraysrailing comma would be good here.

Reply:
updated